### ISSUE TITLE
Works randomly

### ISSUE THREAD

**Issue by beyond9thousand** [Neko-Life/Discord-Screen-Share-Fix-Audio-on-Linux#8]
I was using your commands for so long on KDE... and then even on two window manager installations. Suddenly since the last installation (exact install as a wm i used earlier), it works only at random sessions. Any idea what could be going wrong here? Really wish to debug this

**Comment by beyond9thousand** [comment]
```
local/pipewire 1:0.3.52-2
    Low-latency audio/video router and processor
local/pipewire-alsa 1:0.3.52-2
    Low-latency audio/video router and processor - ALSA configuration
local/pipewire-jack 1:0.3.52-2
    Low-latency audio/video router and processor - JACK support
local/pipewire-pulse 1:0.3.52-2
    Low-latency audio/video router and processor - PulseAudio replacement
local/pipewire-zeroconf 1:0.3.52-2
    Low-latency audio/video router and processor - Zeroconf support
local/wireplumber 0.4.10-3
    Session / policy manager implementation for PipeWire
```

These are the packages on my system

**Comment by Neko-Life** [comment]
This script is for PulseAudio, why are you listing PipeWire modules? What distro are you using? What wm? Anything you suspect that might be the cause?

**Comment by beyond9thousand** [comment]
> This script is for PulseAudio, why are you listing PipeWire modules? What distro are you using? What wm? Anything you suspect that might be the cause?

That is true, but I started playing with loopback modules way back when i first installed manjaro and quickly found out that they work the same irrespective of what you're using.

I'm currently running bspwm, arch Linux
